src/encoded/static/libs/ui/node_graph.js

```diff
@@ -0,0 +1,254 @@
+                clickedCell.attr('class', `${internalSelectedNodes.indexOf(clickedName) > -1 ? 'active-cell' : ''} js-cell-${clickedName} js-cell ${d._children ? 'parent-cell' : ''}`);
+                e.stopPropagation();
+            });
+
+        const nodeGroup = nodeEnter.append('g')
+            .attr('class', (d) => `js-cell-${nodeKeyName(d.data.name)} js-cell ${internalSelectedNodes.indexOf(nodeKeyName(d.data.name)) > -1 ? 'active-cell' : ''} ${d._children ? 'parent-cell' : ''}`)
+            .on('mouseover', (e, d) => {
+                // add hover styles on node
+                d3.select(`.js-cell-${nodeKeyName(d.data.name)}`).selectAll('ellipse').style('transform', 'scale(1.2)');
+                d3.select(`.js-cell-${nodeKeyName(d.data.name)}`).selectAll('ellipse').attr('class', `hover-class ${d.data.class ? d.data.class : 'default'}`);
+                // highlight corresponding matrix row
+                d3.select(`.${d.data.name.replace(/\s/g, '_').toLowerCase()}`).selectAll('th').attr('class', d.data.class);
+            })
+            .on('mouseout', (e, d) => {
+                // remove hover styles from node
+                d3.select(`.js-cell-${nodeKeyName(d.data.name)}`).selectAll('ellipse').style('transform', 'scale(1)');
+                d3.select(`.js-cell-${nodeKeyName(d.data.name)}`).selectAll('ellipse').attr('class', (d.data.class ? d.data.class : 'default'));
+                // remove highlight from corresponding matrix row
+                d3.select(`.${d.data.name.replace(/\s/g, '_').toLowerCase()}`).selectAll('th').attr('class', '');
+            });
+
+        ellipseSettings.forEach((ellipseSetting) => {
+            nodeGroup.append('ellipse')
+                .attr('cx', ellipseSetting.cx)
+                .attr('cy', ellipseSetting.cy)
+                .attr('rx', ellipseSetting.rx)
+                .attr('ry', ellipseSetting.ry)
+                .style('stroke', ellipseSetting.stroke)
+                .style('stroke-width', 1)
+                .attr('class', (d) => (d.data.class ? d.data.class : 'default'));
+        });
+
+        nodeEnter.append('text')
+            .attr('dy', '.35em')
+            .attr('y', (d) => ((d.children && (d.data.name.length < characterLimitForWrap || d.data.name.split(' ').length === 1)) ? offset : d.children ? doubleOffset : 30))
+            .style('text-anchor', 'middle')
+            .text((d) => d.data.name)
+            .attr('class', 'node-text');
+
+        nodeEnter.selectAll('text')
+            .call(wrap, textWrapWidth);
+
+        nodeEnter.append('text')
+            .attr('class', (d) => `node-expander-collapser clicker-${nodeKeyName(d.data.name)}`)
+            .text((d) => (d._children ? circlePlus : d.children ? circleMinus : ''))
+            .attr('y', '-10px')
+            .attr('x', '-10px')
+            .style('font-size', '28px')
+            .on('click', function clickFunction(e, d) {
```

Comment:
After the sprint, if interested, let's have a philosophical discussion on this. In what you did here, the function context (this)  is not the "this" outside clickFunction; which new developers who grew up on arrow function may not realize.